boxsdk/client/client.py

```diff
@@ -505,6 +505,7 @@ def get_groups(self, name=None, limit=None, offset=None, fields=None):
         return LimitOffsetBasedObjectCollection(
             url=url,
             session=self._session,
+            additional_params=additional_params,
```

Comment:
this doesn't seem to fit with the rest of the PR

Reply:
It's a bugfix, discovered while adding test coverage for this method.